README.rst

```diff
@@ -166,6 +167,13 @@ By default, rshell uses ANSI color escape codes when displaying the
 prompt and ls output. This option allows colorized output to be
 disabled.
 
+--binary
```

Comment:
I'd be inclined to call this --ascii or --encode. I would expect --binary to send binary data, which is the opposite of what this does.